reflect/src/test/java/io/spine/reflect/InvokablesTest.java

```diff
@@ -90,9 +86,9 @@ void resetMethod() {
         @Test
         @DisplayName("set accessible and invoke successfully")
         void allowToSetAccessibleAndInvoke() {
-            var result = setAccessibleAndInvoke(privateMethod, target);
+            var result = Invokables.setAccessibleAndInvoke(privateMethod, target);
```

Comment:
I am not sure in un-importing of any of these static methods, which in general looks like an automated IDEA "help" applied.

Please review.